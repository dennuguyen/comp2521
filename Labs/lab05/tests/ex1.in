b 0 5
i 4 5
r 4 5
C
q
