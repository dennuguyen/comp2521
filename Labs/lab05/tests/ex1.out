Graph has V=6 and E=9
V    Connected to
--   ------------
0    1 4 5
1    0 2
2    1 3 4
3    2 4 5
4    0 2 3 5
5    0 3 4

> x=0   q=1>4>5
x=1   q=4>5>2
x=4   q=5>2>2>3>5
x=5   q=2>2>3>5>3
x=2   q=2>3>5>3>3
x=3   q=5>3>3

Graph has V=6 and E=9
V    Connected to
--   ------------
0    1 4 5
1    0 2
2    1 3 4
3    2 4 5
4    0 2 3 5
5    0 3 4

> 
Graph has V=6 and E=9
V    Connected to
--   ------------
0    1 4 5
1    0 2
2    1 3 4
3    2 4 5
4    0 2 3 5
5    0 3 4

> 
Graph has V=6 and E=8
V    Connected to
--   ------------
0    1 4 5
1    0 2
2    1 3 4
3    2 4 5
4    0 2 3
5    0 3

> Component 0 has 0 1 2 3 4 5

Graph has V=6 and E=8
V    Connected to
--   ------------
0    1 4 5
1    0 2
2    1 3 4
3    2 4 5
4    0 2 3
5    0 3

> 